Homework4/task2.js

```diff
@@ -0,0 +1,9 @@
+function validate(age) {
```

Comment:
1. function should return boolean value but not print it
2. `validate` does not represent what does the function actually do. Use more informative name like `identifyIfAdult`